src/modules/corp-type-module/corp-type-module.ts

```diff
@@ -101,7 +101,7 @@ const CorpInfoArray: Array<CorpInfoIF> = [
     colinInd: true,
     corpClass: CorpClass.BC,
     shortDesc: 'BENEFIT COMPANY',
-    fullDesc: 'Benefit Company',
+    fullDesc: 'BC Benefit Company',
```

Comment:
Thanks.

I guess multiple dependencies will need to be updated once this change is published. Please keep a todo list (or create tickets for that).

Reply:
Definitely 👍